CMakeLists.txt

```diff
@@ -133,12 +133,14 @@ option(NGRAPH_CODE_COVERAGE_ENABLE "Enable code coverage data collection" FALSE)
 option(NGRAPH_LIB_VERSIONING_ENABLE "Enable shared library versioning" FALSE)
 option(NGRAPH_PYTHON_BUILD_ENABLE "Enable build nGraph python package wheel" FALSE)
 option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" ${PLAIDML_FOUND})
+option(NGRAPH_IE_ENABLE "Enable the Inference Engine backend" FALSE)
```

Comment:
Could you use `NGRAPH_UNIT_TEST_OPENVINO_ENABLE` to enable IE backend?

or the second variant is removing `NGRAPH_UNIT_TEST_OPENVINO_ENABLE` option and replace this to `NGRAPH_IE_ENABLE`

Also please notify if cmake options should be updated (for example NGRAPH_IE_ENABLE ON for the Inference Engine).

Reply:
@ilyachur Hi, I've remove the `NGRAPH_UNIT_TEST_OPENVINO_ENABLE` flag from our cMake. The `NGRAPH_IE_ENABLE` which is used to enable this backend should be enough. Please update your CMake.

Reply:
Hi @arogowie-intel,

Thank you for the notification, but we will be able to update CMake files only when update nGraph repository to new master which will contain these changes. Because in other case CMake will ignore this option and old tests also will be disabled.

Reply:
OK, this shouldn't be a problem. We simply pass `-DNGRAPH_IE_ENABLE` flag when running cmake command in DLDT and it works.